examples/store/app/products/pages/products/infinite.tsx

```diff
@@ -22,7 +22,7 @@ const Products = () => {
       ))}
 
       <div>
-        <button onClick={() => fetchMore()} disabled={!canFetchMore || isFetchingMore}>
+        <button onClick={() => fetchMore()} disabled={!canFetchMore || !!isFetchingMore}>
```

Comment:
Why this change?

Reply:
This is because the type of `disabled` is a boolean but the type of `isFetchingMore` has been changed from `boolean` to `false | 'previous' | 'next'`.